packages/vue-app/template/client.js

```diff
@@ -501,6 +517,11 @@ async function render (to, from, next) {
       this.$loading.manual = Component.options.loading === false
 
       <% if (features.fetch) { %>
+        <% if (isFullStatic) { %>
+        if (!this.$isPreview) {
+          promises.push(this.fetchPayload(to.path).catch(err => null))
```

Comment:
Why are we hiding error?

Reply:
If an error occurs, it is because we did not generated the payload and should go back to normal fetch behaviour (SPA fallback case)

Reply:
But it might be also an internal error with `fetchPayload` or network request. Maybe we can add reason as a comment above and mention in docs about this fallback behaviour on any network issue? SPA fallback means api call?

Reply:
Adding a comment